hydra_base/lib/users.py

```diff
@@ -22,10 +22,11 @@
 
 from .objects import JSONObject
 from ..db.model import User, Role, Perm, RoleUser, RolePerm
-from ..exceptions import ResourceNotFoundError, HydraError
+from ..exceptions import ResourceNotFoundError, HydraError, HydraLoginUserNotFound, HydraLoginUserMaxAttemptsExceeded, HydraLoginUserPasswordWrong
 from .. import db
 from .. import config
 from ..util.permissions import required_perms
+import transaction
```

Comment:
Hydra base does not use transactions in this way. Transactions should be managed using the 'commit_transaction' function in db/__init__.py. 

We never commit transactions inside functions in this way.

Reply:
I found in make_user_root so I used it. 
But I am going to use commit_transaction

Reply:
Ok, Please remove it from there also.

Reply:
Done. What are the valid test? Tests or unittests?